docs/Contribute-to-AlmaLinux-Build-System.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'AlmaLinux Build System'
+---
+
+###### last updated: 2024-09-20
+
+# Contribute to AlmaLinux Build System
+
+The AlmaLinux Build System is managed by the [SIG/Build System](/sigs/Build-System) SIG. 
+
+The project is open for community contributions. If you are interested in contributing to the Build System project, we recommend you start by checking the AlmaLinux Build System project [milestones](https://github.com/AlmaLinux/build-system/milestones) for ongoing tasks and following the guidelines below.
+
+When you are ready to create a pull request with your suggested changes, please follow the **[commit guidelines](#commit-guidelines)** to keep our commit messages consistent across all repositories around the AlmaLinux Build System. 
+
+## Contribute to the codebase
+
+To contribute to development we recommend deploying the AlmaLinux Build System [locally](https://github.com/AlmaLinux/build-system/wiki/Deploy-on-local-and-remote-machines) and following these guide steps for possible modifications and commit guidelines.
+
+### Help Wanted
+
+We are seeking contributors to help us with:
+* Python and JavaScript Developers are needed to improve UI/UX.
+* Add the ability to run test VMs in different clouds (AWS, Azure, etc.).
+* Add the ability to use external repositories for testing scenarios (e.g. LTP for kernel, CentOS tests for different packages, openQA, etc.).
+* Add OpenStack backend driver support.
+* Add Azure backend driver support.
+* Improve ability to delete a build.
+* Add Kubevirt backend support.
+
+#### A new service's config
```

Comment:
I'm not sure if it makes sense to have this and next 2 sections.

In addition to the `help wanted` section, do you think that we can add a list of our git repos?
Yes, I know that they are already in [build-system repo](https://github.com/AlmaLinux/build-system?tab=readme-ov-file#development), but I guess that we can add the most relevant ones here, let's say web-server, frontend, node, sign-node, albs-deploy and alts

Reply:
These sections have been there from the very first version of contributions guidelines, if you confirm they are not relevant and needed anymore, I'm okay with removing these :) 

As for the help wanted part - Alexandra consulted the build system team and helped me update the items. We also have the fresh repos list on the Build System SIG page, but I'd prefer having them listed along with the updated structure scheme. Would you like to help me with it?

Reply:
That was the plan to add that suggestion, but I was also interested in this section. Thanks for confirming this, I'll delete the info :)